src/TestUtils/src/TestUtils.Appium/TestUtils.Appium.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Appium.WebDriver" Version="5.0.0-beta02" />
```

Comment:
How will this work on Windows given WinAppDriver doesn't work with Appium.WebDriver >= 5.0.0-beta01?

https://github.com/appium/dotnet-client/issues/487

Reply:
this is working you can try it yourself.. I will add info on the description